.circleci/config.yml

```diff
@@ -103,6 +171,13 @@ workflows:
       - test:
           requires:
             - install-dependencies
+      - install-e2e-dependencies:
+          requires:
+            - install-dependencies
+      - e2e-test:
```

Comment:
Is this reasonable to run on every build? Or should we put it behind a hold step?

Reply:
I think it's reasonable to run it before merging a pull request, so probably every time when a PR is open. I know it currently takes much time to run, so putting it behind a button could be a intermediate measure before speeding it up.

Reply:
Maybe I'll put it behind the deploy-to-production approval just for simplicity?

Reply:
Would seem reasonable for now.